src/index.js

```diff
@@ -30,6 +31,7 @@ ReactDOM.render(
       statement="Low for the rest of the day" 
 
     />
+    <Bnglore></Bnglore>
```

Comment:
Instead of individual city name, create common componet

<City name="Banglore" temprature="20" ...otherProp />